### PR TITLE
Add a basic page to display confirmed users' other statuses

### PR DESCRIPTION
**Pull request by alvin319**

### CODE REVIEW

File: pepper/templates/users/additional_status.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                    <li>
+                        Proof of receipts is required and also submit a project on Devpost during HackTX.
+                    </li>
+                    <li>
+                        You can pick up your reimbursement on Sunday, October 29 at HackTX. It will be in the form of an
+                        Amazon gift card.
+                    </li>
+                </ul>
+            {% endif %}
+            {% if campus_ambassador_status == False %}
```
Comment (rjchee) [inline pepper/templates/users/additional_status.html]: You could use `{% if not campus_ambassador_status %}`

File: pepper/users/views.py [hunk 2]
```diff
@@ -444,6 +444,30 @@ def sign():
         return redirect(url_for('dashboard'))
 
 
+@login_required
+@user_status_whitelist(status.CONFIRMED)
+def additional_status():
```
Comment (rjchee) [inline pepper/users/views.py]: I think we shouldn't allow access to this page if they didn't apply for campus ambassador or travel reimbursement

File: pepper/templates/users/dashboard/confirmed.html [hunk 3]
```diff
@@ -5,6 +5,8 @@
         <div class="large-text">
             <p>Thanks for confirming to go to {{ config.HACKATHON_NAME }} {{ user.fname }}! We can't wait to see you on
                 October 28nd!</p>
+            <p>Click <a href="{{ url_for('additional-status') }}">here</a> to find out about your campus ambassador or
```
Comment (rjchee) [inline pepper/templates/users/dashboard/confirmed.html]: Guard with an if block to hide from users this is irrelevant to

File: pepper/templates/users/additional_status.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
+{% extends 'layouts/user_base.html' %}
+{% block content %}
+    <div class="container">
```
Comment (rjchee) [inline pepper/templates/users/additional_status.html]: For this container and the other container, only render it it's relevant to them

File: pepper/users/views.py (hunk 2 above)
Comment (alvin319) [reply pepper/users/views.py]: I think I can just hide it in the jinja template so we can keep this the same?

File: pepper/templates/users/dashboard/confirmed.html (hunk 3 above)
Comment (alvin319) [reply pepper/templates/users/dashboard/confirmed.html]: ok i got it working

**Comment by rjchee** [comment]
This has been merged

**Pull request closed without merging**